Specs/DateTime/Dutch/TimePeriodParser.json

```diff
@@ -1,14 +1,13 @@
 [
   {
-    "Input": "Ik ben 5 tot 18:00 weg",
```

Comment:
Do we have this case already repeated? If not, please keep both the original one and the new one in the specs.

Reply:
ok, original case reinserted